### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dissom**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,62 @@
...
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> "Distance":
```
Comment (vasylhnatiuk) [inline app/main.py:12]:
```suggestion
    def __add__(self, other: int | float | Distance) -> Distance:
```

Comment (vasylhnatiuk) [reply app/main.py:12]: use without "

Comment (vasylhnatiuk) [reply app/main.py:12]: the same everywhere

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+
+    def __ge__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        elif isinstance(other, (int, float)):
+            return self.km >= other
+
+    def __truediv__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)):
+            if other != 0:
```
Comment (vsmutok) [inline app/main.py:64]:
```suggestion
        if isinstance(other, (int, float)) and other != 0:
```

Comment (vsmutok) [reply app/main.py:64]: it is better to write "and" instead of two "if"

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+
+    def __ge__(self, other: int | float) -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        elif isinstance(other, (int, float)):
+            return self.km >= other
+
+    def __truediv__(self, other: int | float | Distance) -> Distance:
+        if isinstance(other, (int, float)) and other != 0:
+                return Distance(round((self.km / other), 2))
```
Comment (vsmutok) [inline app/main.py]:
```suggestion
        if isinstance(other, (int, float)) and other != 0:
            return Distance(round((self.km / other), 2))
```

**Review by vsmutok — CHANGES_REQUESTED** [review]
please fix tests

**Review by vsmutok — APPROVED** [review]
Well Done!

**Pull request closed without merging**